hydradx/tests/test_indexer_utils.py

```diff
@@ -2,14 +2,16 @@
 import math
 
 import pytest
+from hydradx.model.amm.omnipool_amm import OmnipoolState
 from hypothesis import given, strategies as st, reproduce_failure
 from mpmath import mp, mpf
 
 import os
 os.chdir('../..')
 
-from hydradx.model.indexer_utils import get_latest_stableswap_data, get_stablepool_ids, get_fee_history, get_executed_trades, get_stableswap_liquidity_events
-
+from hydradx.model.indexer_utils import get_latest_stableswap_data, get_stablepool_ids, get_omnipool_liquidity, \
+    get_omnipool_asset_data, get_current_block_height, get_asset_info_by_ids, get_current_omnipool, \
+    get_current_omnipool_router, get_fee_history, get_fee_history, get_executed_trades, get_stableswap_liquidity_events
```

Comment:
only need to import get_fee_history once

Reply:
removed